EnviroMonitorWeb/api/models.py

```diff
@@ -72,15 +92,31 @@ def get_update_url(self):
         return reverse('api_metering_update', args=(self.id,))
 
 
+class MeasurmentHistory(Measurment):
+    """
+    Model that holds history entries calculated from Measurment entries. Resolution 1 hour.
+
+    Idea:
+    * background job runs once per hour
+    * job calculates avarage of readings from Measurment entries older than 2 weeks (mean
+    should be calulated per hour basis, one MeasurmentHistory entry per hour)
+    * entries that was used to calculate avarage should be cleaned from Measurment database
+    """
+    pass
+
+
+# do we actually need this one?
+# what is the rationale? grouping Stations? sharing? I would skip it for now...
 class Project(models.Model):
```

Comment:
@zen can we factor out Project model? What is the rationale? Grouping Stations? Sharing? I would skip it for now... Is it necessary?